src/main/webapp/js/adminActivityLog.js

```diff
@@ -3,6 +3,18 @@
 $(document).ready(function() {
     $('#filterReference').toggle();
     AdminCommon.bindBackToTopButtons('.back-to-top-left, .back-to-top-right');
+
+    $('#button_older').on('click', function(e) {
+        e.preventDefault();
+        var nextEndTime = $(e.target).data('nextEndTime');
+        getOlderLogEntriesByAjax(nextEndTime);
+    });
+
+    $('#logsTable tbody tr a').on('click', function(e) {
```

Comment:
`'#logsTable tr a'` should be sufficient. Generally we don't want to have too long of a selector due to performance issues.